examples/live_tiffs.py

```diff
@@ -0,0 +1,129 @@
+"""Loads and Displays tiffs as they get generated in the specific directory.
+Trying to simulate the live display of data as it gets acquired by microscope.
+This script should be run together with live_tiffs_generator.py
+"""
+
+import os
+import sys
+import time
+import numpy as np
+from skimage.io.collection import alphanumeric_key
+from dask import delayed
+import dask.array as da
+from tifffile import imread
+import napari
+from napari.qt import thread_worker
+
+
+with napari.gui_qt():
+    """Starts Napari viewer with in which to display the tiffs.
+
+    Parameters
+    ----------
+    input_dir : str, optional
+        directory to monitor and load tiffs as they start appearing.
+        If no argument is given the then it starts monitoring currect dir.
+    """
+    viewer = napari.Viewer(ndisplay=3)
+    # pass a directory to monitor or it will monitor current directory.
+    path = sys.argv[1] if len(sys.argv) > 1 else '.' 
+    path = os.path.abspath(path)
+    end_of_experiment = 'final.log'
+
+    def append(delayed_image):
+        """Appends the image to viewer.
+        
+        Parameters
+        ----------
+        delayed_image : dask.delayed function object
+        """
+        if delayed_image is None:
+            return
+
+        if viewer.layers:
+             # first run, no layer added yet
+            layer = viewer.layers[0]
```

Comment:
@zeroth you've got the comments backwards now (see my suggestion "if **not** viewer.layers"). Now it's like "no layers added — grab layers[0]" 😂